server/bin/unittests

```diff
@@ -455,7 +470,20 @@ function _setup_state {
     fi
 
     # Setup per-test Dataset DB
+    local _state_db
     _state_db=${_tdir}/state/${1}.db
+
+    local users
+    if [ -f ${_state_db} ]; then
+        users=$(cat ${_state_db} | cut --d ' ' -f 4 | sort -u)
+    else
+        users=
+    fi
+    users="${users} pbench quarantiner satellite"
```

Comment:
Alternatively,
```
    local users = "pbench quarantiner satellite"
    if [ -f ${_state_db} ]; then
        users="$(cat ${_state_db} | cut --d ' ' -f 4 | sort -u) ${users}"
    fi
```

Reply:
Well, you're not wrong, although I'm going to tire of re-running the whole CLI gold suite repeatedly. ;-)

Reply:
And I'll eventually tire of reviewing the same PR repeatedly....  ;-)

(It's only a suggestion -- you've got your "Ship it!" from me -- but it _is_ nicer....  :-D )